src/main/java/com/segment/analytics/android/integrations/adobeanalytics/AdobeIntegration.java

```diff
@@ -513,6 +515,21 @@ private void trackVideo(String eventName, Properties properties) {
       case "Video Playback Seek Completed":
         heartbeat.trackEvent(MediaHeartbeat.Event.SeekComplete, null, null);
         break;
+
+      case "Video Ad Started":
+        MediaObject adInfo =
+            MediaHeartbeat.createAdObject(
+                properties.getString("title"),
+                properties.getString("assetId"),
+                properties.getLong("position", 0),
```

Comment:
Are we considering this in our heartbeat logic? i.e. taking into account the position passed in at the time the event is triggered, and continuing the heartbeat increments based on that

Reply:
Yep, that comes in a later PR.